### PR TITLE
Update comments.json

### PR DESCRIPTION
**Pull request by alexrodba**
Corrected certain inaccuracies. Added coherence to infinitives for actions (save -> desar) and added blank translations.

Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [x] There's a clear use-case for this code change, explained below
- [x] Commit message has a short title & references relevant issues
- [x] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

### CODE REVIEW

**Review by jordevo — APPROVED** [review]

File: ghost/i18n/locales/ca/comments.json [hunk 1]
```diff
@@ -3,69 +3,69 @@
...
-    "Start the conversation": "Comença la conversa",
-    "This comment has been hidden.": "Aquest comentari s'ha amagat.",
+    "Sort by": "Ordenar per",
+    "Start the conversation": "Començar la conversa",
+    "This comment has been hidden.": "Aquest comentari s'ha ocultat.",
     "This comment has been removed.": "Aquest comantari s'ha esborrat.",
-    "Upgrade now": "Millora ara",
+    "Upgrade now": "Actualitzar ara",
     "Yesterday": "Ahir",
     "Your request will be sent to the owner of this site.": "La vostra sol·licitud s'enviarà al propietari d'aquest lloc."
```
Comment (cathysarisky) [inline ghost/i18n/locales/ca/comments.json:70]: Vostra is inconsistent with the informal language used elsewhere, no?

File: ghost/i18n/locales/ca/comments.json [hunk 2]
```diff
@@ -3,69 +3,69 @@
...
-    "Show 1 more reply": "Mostra 1 resposta més",
-    "Show 1 previous comment": "Mostra 1 comentari previ",
-    "Show comment": "Mostra el comentari",
-    "Sign in": "Inici de sessió",
+    "Show {{amount}} more replies": "Mostrar {{amount}} respostes més",
+    "Show {{amount}} previous comments": "Mostrar {{amount}} comentaris previs",
+    "Show 1 more reply": "Mostrar 1 resposta més",
+    "Show 1 previous comment": "Mostrar 1 comentari previ",
+    "Show comment": "Mostrar comentari",
+    "Sign in": "Iniciar sessió",
```
Comment (cathysarisky) [inline ghost/i18n/locales/ca/comments.json:62]: another question about imperative vs infinitive, if you could please confirm.  Context:  These are a labels for a link or a button.

File: ghost/i18n/locales/ca/comments.json [hunk 3]
```diff
@@ -3,69 +3,69 @@
...
+    "Deleting": "Esborrant",
     "Discussion": "Debat",
     "Edit": "Editar",
     "Edit this comment": "Editar aquest comentari",
     "edited": "editat",
     "Enter your name": "Escriu el teu nom",
-    "Expertise": "Conéixement",
+    "Expertise": "Expertesa",
     "Founder @ Acme Inc": "Fundador @ Acme Inc",
     "Full-time parent": "Pare a temps complert",
```
Comment (cathysarisky) [inline ghost/i18n/locales/ca/comments.json:30]: Not your change, but should it be complet or complert?

**Review by cathysarisky — CHANGES_REQUESTED** [review]
Thanks for your work on these - just a few questions!

File: ghost/i18n/locales/ca/comments.json [hunk 4]
```diff
@@ -3,69 +3,69 @@
...
+    "Show comment": "Mostrar comentari",
+    "Sign in": "Iniciar sessió",
     "Sign up now": "Registrar-se ara",
-    "Sort by": "",
-    "Start the conversation": "Comença la conversa",
-    "This comment has been hidden.": "Aquest comentari s'ha amagat.",
+    "Sort by": "Ordenar per",
+    "Start the conversation": "Començar la conversa",
+    "This comment has been hidden.": "Aquest comentari s'ha ocultat.",
     "This comment has been removed.": "Aquest comantari s'ha esborrat.",
```
Comment (jordevo) [inline ghost/i18n/locales/ca/comments.json:67]: not your change but this _comantari_ (instead of _comentari_) hurts my eyes 😉

Comment (alexrodba) [reply ghost/i18n/locales/ca/comments.json:67]: True! let me fix it!

File: ghost/i18n/locales/ca/comments.json (hunk 1 above)
Comment (alexrodba) [reply ghost/i18n/locales/ca/comments.json:70]: You're right. I somehow missed this one. Should be changed to "teva". Thanks for spotting it!

File: ghost/i18n/locales/ca/comments.json (hunk 2 above)
Comment (alexrodba) [reply ghost/i18n/locales/ca/comments.json:62]: I'll change them to imperative as per Softcatala's instructions: https://www.softcatala.org/forum/viewtopic.php?f=4&t=269 although Imperative might be preferred for certain generic actions like seach or sign in. We will keep imperative for specific actions like "show 1 more reply".

File: ghost/i18n/locales/ca/comments.json (hunk 3 above)
Comment (alexrodba) [reply ghost/i18n/locales/ca/comments.json:30]: Complet!

**Review by alexrodba — COMMENTED** [review]
Added changes following suggestions

**Review by cathysarisky — APPROVED** [review]
Looks good to me! Thank you for your work!

**Pull request merged**